wordpress-mobile/WordPress-MediaPicker-Android
# Remove Jetifier

This PR replaces the Tenor library dependency reference with a jetified AAR version.

**To test:**
Verify the library and the sample app can be built and that the GIF search works when running the sample app (you'll need an API key)
